contrib/coredns/policy/README.md

```diff
@@ -1,94 +1,102 @@
+Option **dnstap** defines the attributes to be included in extra field of DNStap message if those attributes are available. The option can be used repeatedly with different values of *LEVEL* parameter to define different sets of dnstap attributes. Value of *LEVEL* should be in range [0..2]. The level of dnstap logging is defined by PDP response with `log` attribute. The default value for log level is 0.
 
-Option log enables log PDP request and response
+The *ATTR_N* parameters have the syntax like `<name>[=<value>]` where value can be one of quoted string, number or IP address. See examples below.
 
-Option max_request_size sets maximum buffer size in bytes for serialized request. Too high limit makes the plugin to allocate too much memory while too small can lead to buffer overflow errors on validation. If "auto" is set plugin allocates required amount of bytes for each request. In case of both "auto" and SIZE, SIZE doesn't limit request buffer but used for cache allocations. 
+Option **passthrough** defines set of domain name suffixes. Domain that has one of these suffixes is resolved without validation. Each suffix should have dot at the end.
 
-Option max_response_attributes sets maximum number of attributes expected from PDP. If value is "auto" plugin allocates necessary attribuets for each PDP response.
+Option **debug_query_suffix** enables debug query feature. Debug query returns some debug information about query processing in text representation. *SUFFIX* should have dot at the end. Debug query uses protocol CHAOS and TXT resource record. See below an example of dig command for debug query.
 
-Option cache enables decision cache. TTL default value is 10 minutes. SIZE limits memory cache takes to given number of megabytes. If it isn't provided cache can grow until application crashes due to out of memory.
+~~~ txt
+dig @127.0.0.1 test.com.debug txt ch
+~~~
+
+Option **debug_id** defines a string to be sent with debug response to identify a CoreDNS instance.
+
+Option **log** enables logging PDP request and response attributes
 
 ## Example
 
 ~~~ txt
 policy {
     endpoint 10.0.0.7:1234, 10.0.0.8:1234
-    edns0 0xffee client_id hex string 32 0 32
-    edns0 0xffee group_id hex string 32 16 32
-    edns0 0xffef uid // equal edns0 0xffef uid hex string
-    edns0 0xffea source_ip ip address
-    edns0 0xffeb client_name bytes string
-    debug_query_suffix debug.
-    debug_id instance_1
     streams 100
-    transfer gid uid
-    dnstap pid
-    metrics uid
+    connection_timeout 1s
+
+    edns0 0xffee client_id hex 32 0 32
+    edns0 0xffee group_id hex 32 16 32
+    edns0 0xffef uid
+    edns0 0xffea client_ip ip
+    edns0 0xffeb client_name bytes
+    validation1 type="query" domain_name
```

Comment:
I understand that, to specify a string like `query`, it *must be* quoted. On the other hand, to specify an integer or IP address, it must *not* be quoted.

I believe this should be made clear in the documentation earlier